tests/api/items/test_items_rest_views.py

```diff
@@ -195,3 +195,16 @@ def test_item_stats(
             {
                 'total': {'checkin': 2},
                 'total_year': {'checkin': 1}}
+
+
+def test_item_stats_endpoint(
```

Comment:
Are we sure this test is correct? Does the fixture "item_at_desk_martigny_patron_and_loan_at_desk" contains a notification operation log that is ignored.